FAILED: akka.pattern.BackoffSupervisorSpec

Timeout in "must support manual reset"

https://jenkins.akka.io:8498/job/akka-multi-node-nightly2/224/consoleFull

https://jenkins.akka.io:8498/job/akka-nightly/2964/consoleFull

```
 must support custom supervision strategy *** FAILED *** (3 seconds, 212 milliseconds)
[info]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsgClass waiting for class akka.pattern.BackoffSupervisor$CurrentChild
[info]   at scala.Predef$.assert(Predef.scala:165)
[info]   at akka.testkit.TestKitBase$class.expectMsgClass_internal(TestKit.scala:435)
[info]   at akka.testkit.TestKitBase$class.expectMsgType(TestKit.scala:408)
[info]   at akka.testkit.TestKit.expectMsgType(TestKit.scala:735)
[info]   at akka.pattern.BackoffSupervisorSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$akka$pattern$BackoffSupervisorSpec$$anonfun$$anonfun$$assertCustomStrategy$1$1.apply$mcV$sp(BackoffSupervisorSpec.scala:85)
[info]   at akka.pattern.BackoffSupervisorSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$akka$pattern$BackoffSupervisorSpec$$anonfun$$anonfun$$assertCustomStrategy$1$1.apply(BackoffSupervisorSpec.scala:82)
[info]   at akka.pattern.BackoffSupervisorSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$akka$pattern$BackoffSupervisorSpec$$anonfun$$anonfun$$assertCustomStrategy$1$1.apply(BackoffSupervisorSpec.scala:82)
[info]   at akka.testkit.TestKitBase$class.poll$2(TestKit.scala:266)
[info]   at akka.testkit.TestKitBase$class.awaitAssert(TestKit.scala:277)
[info]   at akka.testkit.TestKit.awaitAssert(TestKit.scala:735)
[info]   at akka.pattern.BackoffSupervisorSpec$$anonfun$1$$anonfun$apply$mcV$sp$4.akka$pattern$BackoffSupervisorSpec$$anonfun$$anonfun$$assertCustomStrategy$1(BackoffSupervisorSpec.scala:82)
[info]   at akka.pattern.BackoffSupervisorSpec$$anonfun$1$$anonfun$apply$mcV$sp$4$$anonfun$apply$mcV$sp$5.apply$mcV$sp(BackoffSupervisorSpec.scala:100)
```

Similar, but not the same:
```
[info] - must support manual reset *** FAILED *** (3 seconds, 217 milliseconds)
[info]   java.lang.AssertionError: assertion failed: timeout (3 seconds) during expectMsg while waiting for RestartCount(1)
[info]   at scala.Predef$.assert(Predef.scala:165)
[info]   at akka.testkit.TestKitBase$class.expectMsg_internal(TestKit.scala:350)
[info]   at akka.testkit.TestKitBase$class.expectMsg(TestKit.scala:327)
[info]   at akka.testkit.TestKit.expectMsg(TestKit.scala:735)
```

https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/4822/consoleFull

--- a/akka-actor-tests/src/test/scala/akka/pattern/BackoffSupervisorSpec.scala
+++ b/akka-actor-tests/src/test/scala/akka/pattern/BackoffSupervisorSpec.scala
@@ -133,8 +133,11 @@ class BackoffSupervisorSpec extends AkkaSpec with ImplicitSender {
           watch(c1)
           c1 ! "boom"
           expectTerminated(c1)
-          supervisor ! BackoffSupervisor.GetRestartCount
-          expectMsg(BackoffSupervisor.RestartCount(1))
+
+          awaitAssert {
+            supervisor ! BackoffSupervisor.GetRestartCount
+            expectMsg(BackoffSupervisor.RestartCount(1))
+          }
 
           awaitAssert {
             supervisor ! BackoffSupervisor.GetCurrentChild

